migrate/20210623152412_create_tables.go

```diff
@@ -0,0 +1,366 @@
+package migrate
+
+import (
+	"github.com/go-pg/pg/v10/orm"
+	migrations "github.com/robinjoseph08/go-pg-migrations/v3"
+)
+
+func init() {
+	up := func(db orm.DB) error {
+		_, err := db.Exec(`
+			CREATE TABLE pg_chains (
+                name     TEXT  NOT NULL PRIMARY KEY,
+				tip_hash BYTEA NOT NULL
+			)
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE pg_blocks (
+				hash              BYTEA PRIMARY KEY,
+				parent_hash       BYTEA,
+				height            BIGINT NOT NULL,
+				difficulty_target BYTEA  NOT NULL,
+				cum_work          BYTEA  NOT NULL,
+				status            TEXT   NOT NULL,
+				tx_merkle_root    BYTEA  NOT NULL,
+				timestamp         BIGINT NOT NULL,
+				nonce             BIGINT NOT NULL,
```

Comment:
Hmmm... nonce and  extranonce can definitely exceed the width of an int64. But maybe it's OK because they're casted when fetched? A bit surprising that this works.